packages/cli-e2e/utils/platform.ts

```diff
@@ -0,0 +1,8 @@
+import {HTTPRequest} from 'puppeteer-core';
+
+export function isSearchRequest(request: HTTPRequest) {
+  return request.url().startsWith(
```

Comment:
I would just do a regex match valid for any environment:

```
https:\/\/platform[a-z]?\.cloud.coveo.com\/rest\/search
```

Reply:
I'll do a mass update of url platform validation in CDX-98